assets/hip-blocks/block-stream-merkle.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't the bottom left diagram under Root of Input Merkle Tree the one describing input block items.

I guess the most bottom left Output Block Items Merkle Tree should be renamed to Input Block Items Merkle Tree.

Reply:
I think I understand you.
Yes, the added _label_ beneath the input items subtree should be corrected:
~Output Block Items Merkle Tree~
Input Block Items Merkle Tree

Reply:
I've fixed this